app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -744,7 +744,7 @@ export function allowOnlyNumericInput(event, allowNegative = false) {
 export function cleanNullQueryParams(params) {
   for (const [key, val] of Object.entries(params)) {
     if (val === "undefined" || val === "null") {
-      params[key] = null;
+      delete params[key];
```

Comment:
I hope this won't have any unforeseen consequences 😃 (i.e. that there isn't any code that relied on keys being there to clear some state or something)